### PR TITLE
Firefall Client Error Code Support

### PR DESCRIPTION
**Pull request by roribio**

### CODE REVIEW

File: actions/FirefallClient.js [hunk 1]
```diff
@@ -57,7 +69,7 @@ class FirefallClient {
       return response;
     } catch (error) {
       logger.error('Failed generate request:', error);
```
Comment (askayastha22) [inline actions/FirefallClient.js:71]: Let's change the logger statement to 'Failed generating results' to sound more natural.

File: actions/FirefallClient.js [hunk 2]
```diff
@@ -87,7 +99,7 @@ class FirefallClient {
       return response;
     } catch (error) {
       logger.error('Failed feedback request:', error);
```
Comment (askayastha22) [inline actions/FirefallClient.js:101]: Let's modify the logger statement to `'Failed sending feedback:'` to sound more natural.

File: actions/FirefallClient.test.js
Comment (askayastha22) [inline actions/FirefallClient.test.js:1]: Strangely, I don't see this test running even though 'actions/*.js' is specified in the jest command.

Comment (vtsaplin) [reply actions/FirefallClient.test.js:1]: The tests in actions are broken due to the monorepo setup. I am working on fixing them.

**Comment by roribio** [comment]
All review comments addressed. Ready for another review.

**Review by askayastha22 — APPROVED** [review]

**Comment by basecode** [comment]
Fixed 2 checks from https://github.com/adobe/aem-genai-assistant/issues/168

**Pull request merged**